set.mm

```diff
@@ -382312,19 +382312,23 @@ proved using the inner form (see theorem 9.6 of [Schwabhauser] p. 69
       tgbtwnxfr.e $e |- ( ph -> E e. P ) $.
       tgbtwnxfr.f $e |- ( ph -> F e. P ) $.
       tgbtwnxfr.2 $e |- ( ph -> <" A B C "> .~ <" D E F "> ) $.
-      $( Deduce segment congruence from a triangle congruence.  (Contributed by
+      $( Deduce segment congruence from a triangle congruence.  This is a
+         portion of the theorem that congruent parts of congruent triangles are
+         congruent (CPCTC), focusing on a specific segment.  (Contributed by
```

Comment:
CPCTC stands for "corresponding parts of congruent triangles are congruent" (see https://www.cuemath.com/geometry/cpctc/), not "congruent parts of ...".

Reply:
You're absolutely correct, nice catch. I've fixed the change. I'll separately propose changes to other sections based on #2994, #2984, and #3014.